cmd/psqldef/tests.yml

```
@@ -984,6 +984,17 @@ RemoveDefaultExpression:
     );
   output: |
     ALTER TABLE "public"."users" ALTER COLUMN "key" DROP DEFAULT;
+ChangeDefaultExpressionWithAddition:
+  current: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '1 day'::interval)
+    )
+  desired: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '3 day'::interval)
```

I wonder if this fixes the CI failure:

```suggestion
      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '1 day')
    )
  desired: |
    CREATE TABLE public.foo (
      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '3 day')
```

---

That doesn't work because PostgreSQL wants to re-add the ::interval cast.

I'll upload a fix soon.